dev-support/create-release/do-release.sh

```diff
@@ -66,26 +82,34 @@ function should_build {
   fi
 }
 
-if should_build "tag" && [ $SKIP_TAG = 0 ]; then
+if should_build "tag" && [ "$SKIP_TAG" = 0 ]; then
   run_silent "Creating release tag $RELEASE_TAG..." "tag.log" \
-    "$SELF/release-tag.sh"
-  echo "It may take some time for the tag to be synchronized to github."
-  echo "Press enter when you've verified that the new tag ($RELEASE_TAG) is available."
-  read
+    "$SELF/release-build.sh" tag
+  if is_dry_run; then
+    export TAG_SAME_DRY_RUN="true";
+  else
+    echo "It may take some time for the tag to be synchronized to github."
+    echo "Press enter when you've verified that the new tag ($RELEASE_TAG) is available."
+    read -r
+  fi
```

Comment:
Apparently it was in the original code swiped from the Spark project. I think it would be pretty easy to replace it with a little wait loop; I'll do that.

Reply:
No hurry. Nice-to-have. Can come later too.

Reply:
Done.  See commit db4bab6f57d6 just added.